prime-router/metadata/hl7_mapping/ORU_R01/base/specimen.yml

```diff
@@ -28,7 +28,7 @@ elements:
     resource: '%performerOrganization'
     constants:
       organizationIdFieldPath: '%{hl7SpecimenFieldPath}-2-1'
-      identifier: '%specimen.identifier[0].value'
+      identifier: '%diagnostic[%orderIndex].identifier[0].value'
     schema: ../../common/organization-identifier
```

Comment:
It looks like the identifier Simple Report is sending under specimen is not unique, so grabbing it from diagnostic report.

Reply:
Couple of thoughts:
- Just because the identifier simple report is using is not unique, I'm not sure that will be true for other senders
-  If you take a look at what we're mapping from HL7 into `%diagnostic[%orderIndex].identifier[0].value` vs `%specimen.identifier[0].value` MSH-7, ORC.2, ORC.3, ORC.4, OBR.2, OBR.3, or OBR.26 can be put in the list of identifiers for `DiagnosticReport`. Only SPM.2 can be mapped into the list of identifiers for `Specimen`.

I would advise we use the specimen identifier and keep the mapping as is. We have to consider the HL7 senders as we're building this. Just because the value happened to be the same for SimpleReport doesn't mean that will be true for other senders.

Reply:
Thanks for taking time to review this. Moved this to CA receiver transforms and added some comments.